packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -539,6 +553,25 @@ - (BOOL)hasCameraRollPermission
   return [self.permissionsModule hasGrantedPermission:@"cameraRoll"];
 }
 
+
+- (NSString *)mapPreset:(NSString *)preset API_AVAILABLE(ios(11));
+{
+  if ([preset isEqualToString:@"Passthrough"]) return AVAssetExportPresetPassthrough;
```

Comment:
I think having these mappings in a dictionary would be better and clearer than thousands of `if`s 😜